src/scss/packages/usa-image-example/src/_styles.scss

```diff
@@ -0,0 +1,41 @@
+@use 'uswds-core' as *;
+
+@mixin usa-image-example($color) {
+  img {
+    box-shadow: 0 units(1) 0 0 color($color);
+  }
+  .usa-icon-list__icon {
+    color: color($color);
+  }
+  .usa-icon-list__content {
+    .image-example__do-dont {
+      color: color($color);
+      font-weight: bold;
+    }
+  }
+}
+
+.usa-image-example__figure {
+  margin: 0;
+  display: table;
+}
+
+.usa-image-example__figcaption {
+  margin-top: units(1.5);
+  display: table-caption;
+  caption-side: bottom;
+}
+
+.usa-image-example__img--bordered {
```

Comment:
As a BEM modifier, the `--bordered` suffix here implies to me that this is a variant on some base styles that don't include the bordering. But it's my understanding that we always want to have a border, or at least by default. So I'd suggest removing the modifier.

I also have a slight preference for no abbreviations, but I can also see a case being made for it matching the tag name.

```suggestion
.usa-image-example__image {
```

Reply:
Thanks this matches up well with what I'm thinking/intending. I just posted a review here, but:
- `--bordered` should imply that there's a variant. This is a variant (see comment below).
    - Since it's confusing maybe `--fully-bordered` ? Or some other name?
- I just renamed `img` to `image` that was a holdover from the tag name. Agreed it's better as `image`.

Reply:
![without--bordered](https://github.com/18F/identity-design-system/assets/6818839/82d7c265-bd50-4ad0-98a9-c2175b9f08c0)
![with--bordered](https://github.com/18F/identity-design-system/assets/6818839/26ddda81-3ba5-49d0-bdb4-62fa316c7012)

Reply:
The reason not to give every image a full border is it causes an odd little ledge with dark background images. Also, it's not necessary and might cause spacing issues down the line.
![dark-background-with--bordered](https://github.com/18F/identity-design-system/assets/6818839/83d793d8-b8a0-4de2-9782-f7068657fdfe)

Reply:
Oh! I didn't realize we want to have the option to exclude the border. That makes total sense to have the modifier then. Especially now after you added the base class styles in db1511f.